packages/shared/src/components/profile/devcard/DevCardTwitterCover.tsx

```diff
@@ -4,16 +4,16 @@ import { devCardBoxShadow, DevCardTheme, DevCardType } from './common';
 import { DevCardStats } from './DevCardStats';
 import { DevCardFooter } from './DevCardFooter';
 import Logo from '../../Logo';
-import { useDevCard } from '../../../hooks/profile/useDevCard';
+import { UseDevCard } from '../../../hooks/profile/useDevCard';
 
 interface DevCardTwitterCoverProps {
-  userId: string;
+  data: UseDevCard;
```

Comment:
We only use `devcard` and `coverImage` from the `UseDevCard` type. Maybe it's worth narrowing down to those. See my comment above. Here we could do:
```suggestion
  devcard?: DevCardData;
  coverImage?: string;
```

Reply:
Yes, that's the only thing that can be pulled from that object. So essentially, they are the same. The primary reason for doing this was that the component initially used the hook in itself.

Reply:
Refactored here https://github.com/dailydotdev/apps/pull/2603/commits/532f2c8a2fffe4889f4c537cd0ae7ea711c5b835